src/ostorlab/runtimes/local/models/models.py

```diff
@@ -386,12 +384,13 @@ class AgentGroup(Base):
     )
 
     @staticmethod
-    def create(name: str, description: str) -> "AgentGroup":
+    def create(name: str, description: str, agents: Any) -> "AgentGroup":
         """Persist the agent group in the database.
 
         Args:
-            key: Agent group key.
+            name: Agent group key.
```

Comment:
We should use the key since we don't have the notion of organization.

Reply:
as discussed, there will be name and key both